contributing/ci-docs.txt

```diff
@@ -199,24 +199,24 @@ The branch for the 5.3.x series of the OMERO documentation is develop.
 		#. Pushes the auto-generated changes to
 		   :omedoc_scc_branch:`develop/merge/autogen`
 
-Bio-Formats 5.2.x series
+Bio-Formats 5.3.x series
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The branch for the 5.2.x series of the Bio-Formats documentation is develop.
+The branch for the 5.3.x series of the Bio-Formats documentation is develop.
 
 .. glossary::
 
 	:jenkinsjob:`BIOFORMATS-DEV-latest-docs`
 
 		This job is used to build the develop branch of the Bio-Formats
-		documentation and publish the official documentation for the 5.2.x
+		documentation and publish the official documentation for the 5.3.x
 		release of Bio-Formats
 
 		#. |sphinxbuild|
 		#. |linkcheck|
 		#. If the build is promoted,
-			#. |ssh-doc| :file:`bf-5.2-release.tmp`
-			#. |deploy-doc| http://www.openmicroscopy.org/site/support/bio-formats5.2/
+			#. |ssh-doc| :file:`bf-5.3-release.tmp`
+			#. |deploy-doc| http://www.openmicroscopy.org/site/support/bio-formats5.3/
```

Comment:
This will exist once we go ahead with the m1 release, then the build will be happy again https://ci.openmicroscopy.org/view/Docs/job/CONTRIBUTING-merge-docs/